packages/dds/sequence/api-report/sequence.legacy.alpha.api.md

```diff
@@ -358,14 +351,24 @@ export class SequenceInterval implements ISerializableInterval {
     union(b: SequenceInterval): SequenceInterval;
 }
 
+// @alpha (undocumented)
+export const SequenceInterval: {
+    new (client: Client, start: LocalReferencePosition, end: LocalReferencePosition, intervalType: IntervalType, props?: PropertySet, startSide?: Side, endSide?: Side): SequenceInterval;
+    prototype: SequenceInterval;
+};
+
 // @alpha
-export class SequenceMaintenanceEvent extends SequenceEvent<MergeTreeMaintenanceType> {
-    // @deprecated
-    constructor(
-    opArgs: IMergeTreeDeltaOpArgs | undefined, deltaArgs: IMergeTreeMaintenanceCallbackArgs, mergeTreeClient: Client);
+export interface SequenceMaintenanceEvent extends SequenceEvent<MergeTreeMaintenanceType> {
+    // (undocumented)
     readonly opArgs: IMergeTreeDeltaOpArgs | undefined;
 }
 
+// @alpha (undocumented)
+export const SequenceMaintenanceEvent: {
```

Comment:
this should not be exported

Reply:
at a minimum you've lost the deprecation, i'm also not sure if this is technically non-breaking in all cases. i'd rather just keep the pattern i've already got

Reply:
I think the description can be misinterpreted. This PR does not wholly redo any other PR. This PR is the subset of changes from [test/remove-ISegment-deprecations](https://github.com/microsoft/FluidFramework/tree/test/remove-ISegment-deprecations) that can be merged to main right now and not make any waves for customers. But as much as possible it put the revised patterns in place for internal development. With this change the amount of code that needs to be maintained in a branch waiting for breaking window is minimized.
These changes could have been made with or directly after the original deprecations.

So SequenceMaintenanceEvent must be exported in this stage to retain full compatibility. It is worth noting that there are no type test changes in here and they fully pass our compatibility checks.